hub/src/main/java/ch/prevo/open/hub/nodes/NodeConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package ch.prevo.open.hub.nodes;
+
+class NodeConfiguration {
+
+    private String uid;
```

Comment:
Just to clarify, what would be the UID here? The foundation or does every node have a UID? Since one node should also be able to serve multiple foundations as I understood.

Reply:
At the sprint start, the scope was "Retirement Foundation == Node", to my understanding. This should absolutely be extended.